### PR TITLE
[CNTO-4165] - Use Large Multi Selector in the modal

### PR DESCRIPTION
**Pull request by RitikSrivastava-Plivo**

### CODE REVIEW

File: src/components/Select/index.js [hunk 1]
```diff
@@ -115,43 +115,28 @@ export const Select = React.forwardRef(function Select(
               'contacto-multi-select-modal',
               maxTagModalTitle ? '' : 'contacto-multi-select-modal--no-title',
             ].join(' ')}
-            title={maxTagModalTitle}
+            title={'Hello'}
```
Comment (samuellawerentz) [inline src/components/Select/index.js:118]: Is it always going to be hello?

File: src/components/Select/index.js [hunk 2]
```diff
@@ -115,43 +115,28 @@ export const Select = React.forwardRef(function Select(
...
+                placeholder={placeholder}
+                tagRender={(tagProps) => (
+                  <Tag type="select" disableUppercase closeIcon {...tagProps} />
+                )}
                 dropdownClassName={['sg contacto-select-listbox', dropdownClassName].join(' ')}
                 mode="multiple"
                 value={localValue}
                 onChange={onValueChange}
-                style={{ '--placeHolderText': `"${placeholder}"` }}
+                maxTagCount={null}
```
Comment (samuellawerentz) [inline src/components/Select/index.js:137]: Can we make this a prop? 
And default it to `null` if not found?

**Review by samuellawerentz — CHANGES_REQUESTED** [review]
Please check that other multi select in other sections of the app doesn't break.

File: src/components/Select/index.js (hunk 1 above)
Comment (RitikSrivastava-Plivo) [reply src/components/Select/index.js:118]: No, Sorry
That was a debug statement
Lemme revert

File: src/components/Select/index.js (hunk 2 above)
Comment (RitikSrivastava-Plivo) [reply src/components/Select/index.js]: We don't want it to be anything except null, a better way is to just not pass it, and remove {...props}

Done ✅

**Review by samuellawerentz — APPROVED** [review]

**Pull request merged**